docker/run_tests.bash

```diff
@@ -8,4 +8,8 @@ docker run \
     --network="dbt-net" \
     -v $PWD/dbt:/root/.dbt \
     dbt-trino \
-    "cd /jaffle_shop && dbt seed && dbt run && dbt test"
+    "cd /jaffle_shop \
+        && dbt seed \
+        && dbt run \
+        && dbt run --model dim_order_dates \
```

Comment:
did you miss to add a comment here?

Reply:
if i add the comment, then the next line will not be executed anymore

Reply:
that should work:
```
docker run \
    --network="dbt-net" \
    -v $PWD/dbt:/root/.dbt \
    dbt-trino \
    "cd /jaffle_shop \
        && dbt seed \
        && dbt run \
        && dbt run --model dim_order_dates `# some comment` \
        && dbt test"
```

Reply:
I've inserted the comment before the statement to avoid over complicating the statement.